api/app/main.py

```diff
@@ -155,7 +156,7 @@ async def get_hourlies(request: schemas.StationCodeList, _: bool = Depends(authe
     """ Returns hourlies for the last 5 days, for the specified weather stations """
     try:
         LOGGER.info('/hourlies/')
```

Comment:
`logger` ?

Reply:
I've been slowly making it lower case wherever I find it - but ok - I'll do main too :)